### PR TITLE
PR build test

### PR DESCRIPTION
**Pull request by kkuzmin**
### Solution Description
PR build test

### CODE REVIEW

**Comment by kkuzmin** [comment]
test it

**Comment by kkuzmin** [comment]
test it

File: al_aws_collector.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ class AlAwsCollector {
                         util.inspect(error);
             }
             const status = this.prepareErrorStatus(errorString);
-            this.sendStatus(status, () => {
+            this.sendStatus(status, (err) => {
```
Comment (JDragovichAlertLogic) [inline al_aws_collector.js]: Should we handle that sending error?

Though handling the error handlers' error is making my head hurt.

Comment (kkuzmin) [reply al_aws_collector.js]: Removed `err`

**Comment by kkuzmin** [comment]
closing in favor of https://github.com/alertlogic/al-aws-collector-js/pull/50

**Pull request closed without merging**